packages/editor/src/components/page-attributes/style.scss

```diff
@@ -17,3 +17,7 @@
 		min-width: 320px;
 	}
 }
+
+.editor-post-parent__panel-dialog__example-url {
+	word-break: break-word;
```

Comment:
I didn't mean to be prescriptive. 😄 

Was `<wbr />` not working? 

Having said that, I checked in Firefox, Chrome and Safari and this is looking good to me.

<img width="578" alt="Screenshot 2024-08-06 at 12 57 14 PM" src="https://github.com/user-attachments/assets/0bbb2225-e1ed-48e0-bfa3-9624270a2c91">

Reply:
> Was `<wbr />` not working?

The span is around the homeURL because it doesn't have `wbr` for dots or slashes. Can you think of a way I could add it, because I think that would be nicer than having breaks in the mi
ddle of words.

Reply:
🤔 

Not off the top of my head... unless you play around with the [MDN example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr#examples) 🤷🏻

Reply:
@ramonjd Night brain realised the order of operations allowed me to add the elements to the homeURL, added in c1f8aa7745bcfddc816376c863715835d0e865a8.

| Chrome | Firefox | Safari |
| --- | --- | --- |
| ![chrome](https://github.com/user-attachments/assets/b2795b3f-0d64-42e2-8c45-99b14400551c) | ![firefox](https://github.com/user-attachments/assets/46f10ab9-9969-4022-af16-2d8808b67305) | ![safari](https://github.com/user-attachments/assets/931643b9-fb68-49f5-a9a0-a29e05286b6d) |